cmd/secret/cmd/root.go

```diff
@@ -136,6 +161,8 @@ func (r *root) setCmd() *cobra.Command {
 	setCmd.Flags().StringVarP(&cipherKey, "cipher-key", "c", cipherKey, "cipher key for data encryption and decryption")
 	setCmd.Flags().StringVarP(&path, "path", "p", "file.txt", "the place where the key/value will be stored/got")
 	setCmd.Flags().StringVarP(&redisURL, "redis-url", "r", "", "redis url address. Example: localhost:6379")
+	setCmd.Flags().StringVarP(&postgresURL, "postgres-url", "s", "", "postgres url address. Example: localhost:5432")
```

Comment:
In documentation you mentioned example of postgres url but it will not work because it has no specified database or credentials.